source/gui/settingsDialogs.py

```diff
@@ -2645,9 +2647,60 @@ def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 		)
 		self.bindHelpEvent("ParagraphStyle", self.paragraphStyleCombo)
 
+		# Translators: This is a label for sentence reconstruction in the document navigation dialog
+		label = _("&Sentencew Sentence reconstruction:")
+		self.sentenceReconstructionCombo: nvdaControls.FeatureFlagCombo = sHelper.addLabeledControl(
+			labelText=label,
+			wxCtrlClass=nvdaControls.FeatureFlagCombo,
+			keyPath=["documentNavigation", "sentenceReconstruction"],
+			conf=config.conf
+		)
+		self.bindHelpEvent("SentenceReconstruction", self.sentenceReconstructionCombo)
+
+		language = speech.getCurrentLanguage()
+		prettyLanguage = languageHandler.getLanguageDescription(language)
+		# Translators: This is placeholder for the situation when we couldn't determine current language.
+		# It is used in document navigation settings panel to describe non-breaking prefixes edit box.
+		prettyLanguage = prettyLanguage or _("Unknown language")
+		# Translators: This is the label for a textfield in the
+		# Document Navigation settings panel.
+		label = _("Non-breaking prefixes [%s]")

```

Comment:
Why is it called prefix?

```suggestion
		label = _("Sentence stop exceptions [%s]")
```

or something else similar or even more suitable.